### DISCUSSION TITLE
Custom Target marker is not applying over the link but a little below

### DISCUSSION THREAD

**Discussion by vyombriq** [clientIO/joint#2802] [Q&A]
### Introduction

![image](https://github.com/user-attachments/assets/ab3cd94b-dc9e-4de7-aa03-42df8f6ecb3b)
My custom target marker are not applying on correct position like on the link , it is placed below the image but i see the path coordinates to be the same 

following is the svg code
```
<svg width="200" height="200" xmlns="http://www.w3.org/2000/svg">
  <path id="Vector" d="M8 18C6.33333 18 4.91667 17.4167 3.75 16.25C2.58333 15.0833 2 13.6667 2 12C2 10.3333 2.58333 8.91667 3.75 7.75C4.91667 6.58333 6.33333 6 8 6C9.5 6 10.8042 6.475 11.9125 7.425C13.0208 8.375 13.6917 9.56667 13.925 11H22H13.925C13.6917 14.4333 13.0208 15.625 11.9125 16.575C10.8042 17.525 9.5 18 8 18Z" fill="black"/>
</svg>
```

### Restrictions & Constraints

need answer using standard.link 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
The coordinate system of markers is explained here: https://docs.jointjs.com/api/dia/Paper/#definemarker

You need to change the path to be located around origin `0,0` (alternatively you can keep the `d` attribute as is, but add `transform="translate(-x, -y)" attribute).
